### ISSUE TITLE
pipeline parameters

### ISSUE THREAD

**Issue by andrewrandsbluestone** [MicrosoftDocs/vsts-docs#7118]
Hi, I'm trying to setup a release and it _seems_ to deploy. However, the IIS Web App Manage task has Website name fixed as 'Default Web Site'. The link thing states "This setting is linked to the 'Website name' (Parameters.WebsiteName) pipeline parameter."

I just cannot figure out how to access this Parameters.WebsiteName setting and deploy to a different website.

I notice that here https://github.com/MicrosoftDocs/vsts-docs/issues/2102#issuecomment-521165342

it is stated "Please note, setting process parameters is supported only in build pipelines and not in release pipelines. ". I cannot see the value of setting Website name in a build pipeline. 
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ca1c5394-611b-b119-dbf3-b88404e351d5
* Version Independent ID: fcc23259-21ec-f9fa-4e25-f587e1e0f41b
* Content: [Deploy to IIS on a Windows VM - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/apps/cd/deploy-webdeploy-iis-deploygroups?view=azure-devops#feedback)
* Content Source: [docs/pipelines/apps/cd/deploy-webdeploy-iis-deploygroups.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/pipelines/apps/cd/deploy-webdeploy-iis-deploygroups.md)
* Product: **devops**
* Technology: **devops-cicd**
* GitHub Login: @RoopeshNair
* Microsoft Alias: **ronai**

**Comment by RoopeshNair** [comment]
Website name should be editable, you can type-in. Unless, I'm missing something here. Can you share some screenshots?

**Comment by andrewrandsbluestone** [comment]
In Azure Dev Ops: 

1. Select 'Releases'
2. Select 'New release Pipeline'
3. From Select a template, select 'IIS website deployment (Deployment Group: Deploy an ASP.NET or ASP.NET Core web application to an IIS website on physical or virtual machines (VM).'
4. This creates two tasks,'IIS Web App Manage' & 'IIS Web App Deploy'
5. Open IIS Web App Manage' - Website Name is locked as 'Default Web Site'
![Release pipeline1](https://user-images.githubusercontent.com/58685513/73526733-bc229980-4409-11ea-97e7-db1400e4cbac.png)

**Comment by andrewrandsbluestone** [comment]
I think I have managed to workaround this by creating my own empty Stage.

Selected 'Add a deployment group job'
In Add Tasks, selected 'IIS web app manage' & 'IIS web app deploy' tasks
The 'Manage IISWebsite' task that gets added has an editable Website name

![Release pipeline2](https://user-images.githubusercontent.com/58685513/73527941-d3fb1d00-440b-11ea-8344-8a103afcf20c.png)

**Comment by RoopeshNair** [comment]
You can click on the stage and you will be able to edit the website name

**Comment by andrewrandsbluestone** [comment]
I can't see anywhere at the stage level where the Website Name is visible, let alone editable

**Comment by RoopeshNair** [comment]
![Screenshot_20200131-172251](https://user-images.githubusercontent.com/1370480/73537412-69b79c00-444e-11ea-8beb-267d88abd357.jpg)

You should see website name as in this screenshot. Excuse brevity, on my phone

**Comment by andrewrandsbluestone** [comment]
Looks like you picked a different template - 'IIS website and SQL database deployment'

**Comment by andrewrandsbluestone** [comment]
Why has this been closed? The problem exists with the template I used. I have no reason to use a template that involves SQL Server

**Comment by RoopeshNair** [comment]
I tried with the template you suggested and I'm able to edit. 

![image](https://user-images.githubusercontent.com/1370480/73828836-fc2db600-4827-11ea-85c6-ce50246458bc.png)
Click on the stage 1 and you can edit the text "Default Web Site" to whatever the website name is..

![image](https://user-images.githubusercontent.com/1370480/73828775-e15b4180-4827-11ea-9c7e-d89e77d51fb3.png)

**Comment by kenturley** [comment]
Thank you @andrewrandsbluestone for the workaround. I've used the full template in the past and it allowed me to enter a web site name. Now it doesn't. The only way I could get there was by adapting your solution, except I had to begin by adding via the IIS (without SQL) template since it was the only way I could find to reference my Deployment Group. Then I deleted the template's Manage and Deploy IIS tasks, then I added them back in manually as you indicate. Finally I could specify my web site name.

Absolutely agree: **this issue should not be closed!**  It is still very much an active problem with hard-coded dependency on a parameter, Parameters.WebsiteName, and no indication of how to set it or where it comes from. Super frustrating.
